common/schema-builder/src/test/java/io/smallrye/graphql/index/SchemaBuilderTest.java

```diff
@@ -183,6 +183,23 @@ public void testSchemaWithDirectives() throws IOException {
         assertArrayEquals(new String[] { "getter" }, (Object[]) titleDirectiveInstance.getValue("value"));
     }
 
+    @Test
+    public void testGenericSchemaBuilding() {
+        Indexer indexer = new Indexer();
+        indexDirectory(indexer, "io/smallrye/graphql/index/generic");
+        IndexView index = indexer.complete();
+
+        Schema schema = SchemaBuilder.build(index);
+
+        assertNotNull(schema);
+
+        String schemaString = JSONB.toJson(schema);
```

Comment:
Instead of making very non-specific assertions about the schema string, couldn't we look at the operation arguments and return types?
Call `schema.getQueries()` or `schema.getMutations()`, retrieve operations from that set and then use `operation.getArguments()` for arguments and `operation.getReference()` for the return type?